### PR TITLE
compat version with test

### PR DESCRIPTION
**Pull request by CarlosOrozco88**

### CODE REVIEW

**Comment by r-murphy** [comment]
Hi @CarlosOrozco88 .
Thanks for the PR. I'll have a look at it soon. But I can't merge as-is with the 2 latest commits removing me as a contributor and changing the package name.

Ryan

**Comment by CarlosOrozco88** [comment]
Im so sorry, i thougth the repository was dead and i published the package fork in npm. I will delete it and use your package dependency. 
I'm developing a vscode extension and i need it in order to build for newer ui5 versions.
Thanks!

Edit: i deleted de package ui5-preload from npm. I will wait for changes in this package to publish the extension

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
-
-# openui5-preload
+# ui5-preload
```
Comment (r-murphy) [inline README.md]: @CarlosOrozco88 can you revert the rename in `README.md` and the mention of the fork. Otherwise, looks good to me.

**Review by r-murphy — CHANGES_REQUESTED** [review]

**Review by r-murphy — APPROVED** [review]

**Comment by r-murphy** [comment]
Looks good. Thanks!

**Comment by r-murphy** [comment]
@CarlosOrozco88  I just published version 3.0.0 (major bump due to some deps requiring node 10+). Please give it a try and let me know how it goes. 
Also, are you using the CLI or only the imported API? Since you plan to use this in a VS Code extension, I can pull the CLI out into its own module to minimize dependencies for your extension.

**Pull request merged**